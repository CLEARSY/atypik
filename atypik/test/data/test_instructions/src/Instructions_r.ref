/* Instructions_r
 * Author: vdemeulenaere
 * Creation date: 27/06/2023
 */

REFINEMENT Instructions_r
REFINES Instructions


 END