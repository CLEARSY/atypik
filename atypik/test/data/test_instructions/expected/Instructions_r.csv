Expression	Identifier	Line	Column	Type
